src/constants/index.ts

```diff
@@ -0,0 +1,6 @@
+import { CommunicationMethodV1 } from '@ge-fnm/action-object'
+
+// if the browser is ever capable of making calls to the radio, put the methods here
+export const BROWSER_ENABLED_COMM_METHODS: [CommunicationMethodV1?] = []
```

Comment:
If you are trying to say that this array should be a list of values that are all type A, you would do
`const list: A[] = []`

The way you have it now, [CommunicationMethodV1?] = [] will allow for objects in the array to be undefined. For example this would compile fine:
```
BROWSER_ENABLED_COMM_METHODS = [undefined, CommunicationMethodV1.HTTP]
```

Unless Im missing something, I would suggest that you change ```[CommunicationMethodV1?]``` to ```CommunicationMethodV1[]```

Reply:
Good catch. From the looks of things, we don't want `undefined` comm. methods in this array. You'll probably want to change this, @danielkjones.

Reply:
Will address this in my next commit.